public/javascript/location.js

```diff
@@ -0,0 +1,78 @@
+// dummy number for circle size, cityCount will be per city
```

Comment:
suggest re-naming this file to country-codes.js to help differentiate between files for Location model

Reply:
I can rename; however, this file basically contains logic needed to produce map for the Locations nav bar link.

Let me know if you still feel it needs changed and I can make that change.

Reply:
Hi Kristen, I am going to change this to Resolve conversation but have taken note of this topic.....please reach out if you still feel name should be changed to avoid confusion or we can discuss more on Thursday.